src/phoenix/trace/__init__.py

```diff
@@ -8,3 +11,16 @@
     "DocumentEvaluations",
     "TraceEvaluations",
 ]
+
+
+@contextlib.contextmanager
+def suppress_instrumentation() -> Iterator[None]:
+    """Context manager to pause OpenTelemetry instrumentation."""
+    try:
+        from opentelemetry.context import _SUPPRESS_INSTRUMENTATION_KEY, attach, detach, set_value
+    except ImportError:
+        yield
+        return
+    token = attach(set_value(_SUPPRESS_INSTRUMENTATION_KEY, True))
+    yield
+    detach(token)
```

Comment:
I'm guessing custom spans will still come through? Doesn't have to be a part of this PR but we might want to account for that eventually.

Reply:
nothing we can do for custom spans, because this is an opt-in mechanism

Reply:
we would have to temporarily disable the tracer_provider

Reply:
disabling the tracer_provider is not always possible, because it could be embedded in each instrumentor separately